### ISSUE TITLE
Techniques H42 with contents img element only.

### ISSUE THREAD

**Issue by motchie** [squizlabs/HTML_CodeSniffer#102]
Hi.

When I check the HTML something like:
```
<h1><img src='foo.png' alt='Header' /></h1>
```
with HTML_CodeSniffer in WCAG 2.0 Level A mode, I got message:
```
Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
```
which is in [Standard].Principle1.Guideline1_3.1_3_1.H42.2.

I think it should replace img element with alt text first, then checking the content of H1 element.

**Comment by fedir** [comment]
:+1:

**Comment by ironikart** [comment]
Duplicate issue with #85 and is resolved with c096b64c1085d95072cdada91d787583a30a3a4b.
